Triangle/src/main/java/RectangularTriangle.java

```diff
@@ -0,0 +1,8 @@
+public class RectangularTriangle {
+
+    public static boolean ifRectangularTriangle(float a ,float b,float c){
+
+                return (float) Math.pow(a,2) + (float) Math.pow(b,2) == (float) Math.pow(c,2);
```

Comment:
zamiast 'ifRect[...]' to chyba lepiej 'isRect[...]'. Inny przyklad:

```
 boolean isBusy(Chair chair)
```
vs
```
boolean ifBusy(Chari chair)
```

pozniej wywolanie np. w ifie:
```if(isBusy(chair)) [...]```
vs
```if(ifBusy(chair)) [...]```
;)

Reply:
druga sprawa - skad wiadomo w jakiej kolejnosci podać parametry. Czy nie lepiej zalozyc, ze możemy podać:

isRectangularTriangle(4,3,5) lub (3,4,5) lub (5,4,3) i w każdym przypadku mamy trójkąt prostokątny, a twoja metoda będzie zwracać true/false w zależności od kolejności argumentów. Trochę błędogenne. :P 

Można to sprytnie ogarnąc (bez ifów!), pomyśl jak. ;) Hint: wykorzystaj tablicę i odpowiednią operację na niej.